### PR TITLE
Hover States

### PR DESCRIPTION
**Pull request by tcho6319**
### Summary 

This pull request adds hover states to buttons (i.e. clickable elements) on CP. I globally set buttons to have 0.5 opacity on hover in `src/assets/scss/_global.scss`. This means that every `button` on CP should have a hover state. 

**NOTE: Some buttons like the nav bar icons, course/semester menu items, semester view compact/not compact toggle buttons should have full opacity so I set the `opacity: 1` to those less frequent special cases.**

For Notion item: [Add hover states to buttons](https://www.notion.so/Add-hover-states-to-buttons-ae61c3e48dc145edad9fd69444b69e96)

- Onboarding:
   - [x] Change "Next" & "Finish" button hover states to 50% lighter of the same color (it's currently gray)
   - [x] Change the "< Previous" text to also have 50% lighter hover.
   - [x] Add hover state to the "+ add another major", "edit" text

- Add Semester
   - [x] Cancel and Add buttons. Change text to "Cancel" and "Add". (In general, CP follows this pattern) Add hover states for them. For "Add" make the background color lighter.
   - [ ] For "Cancel" button, only make text lighter **NOTE: I could not figure out how to do this. If anyone knows, pls comment!**

- Edit Semester
   - [x] Cancel and Edit buttons. Add hover states. Please also center text horizontally and vertically.
 
- [x] Changed some "X" imgs to buttons and some clickable divs to buttons

### Test Plan 
Be sure to look through the site and ensure that nothing visually changed (besides the added hover states). Note that many requirements bar dropdown titles, major/minor names have a hover state now because they are all buttons. Lmk if you want them to be full opacity (`opacity: 1`).

### CODE REVIEW

File: src/assets/scss/_global.scss [hunk 1]
```diff
@@ -101,7 +101,7 @@ button {
 }
 
 button:hover {
-  opacity: 0.95;
+  opacity: 0.5;
```
Comment (tcho6319) [inline src/assets/scss/_global.scss:104]: Globally all buttons would by default have 50% opacity on hover.

File: src/components/BottomBar/BottomBar.vue [hunk 2]
```diff
@@ -76,6 +76,9 @@ export default Vue.extend({
     width: calc(100vw - 29.5rem);
     text-align: left;
     padding-left: 0;
+    &:hover {
+      opacity: 1;
+    }
```
Comment (tcho6319) [inline src/components/BottomBar/BottomBar.vue:81]: We want the Bottom Bar Title to have 100% opacity

File: src/components/Modals/Onboarding/Onboarding.scss [hunk 3]
```diff
@@ -1,5 +1,9 @@
 @import '@/assets/scss/_variables.scss';
 
+button:hover {
+  opacity: 0.5;
+}
+
```
Comment (tcho6319) [inline src/components/Modals/Onboarding/Onboarding.scss:6]: Added this since Onboarding modal components use `Onboarding.scss`

**Review by SamChou19815 — APPROVED** [review]
Overall LGTM. But the requirement text might be too light and thus not enough contrast?
Screen Shot 2021-03-27 at 10 38 08

**Comment by tcho6319** [comment]
> Overall LGTM. But the requirement text might be too light and thus not enough contrast?
> Screen Shot 2021-03-27 at 10 38 08

@SamChou19815 I increased the opacity on hover to be 0.8.

**Pull request merged**